.ops/build-docker/Dockerfile

```diff
@@ -0,0 +1,88 @@
+FROM ubuntu:22.04
+
+ARG USER_ID=1001
+ARG GROUP_ID=1001
+
+ENV DEFAULT_JAVA_VER 11
+ENV JAVA_HOME /usr/lib/jvm/java-${DEFAULT_JAVA_VER}-openjdk-arm64/
+ENV DEBIAN_FRONTEND noninteractive
+# ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV JDK_JAVA_OPTIONS "-Xmx512m -Xms512m -XX:MaxDirectMemorySize=700m -XX:ConcGCThreads=4 -XX:ParallelGCThreads=4"
+
+# tzdata ENVs required for installing software-properties-common
+ENV TZ=US/Eastern
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get -qq update && \
+    apt-get install -y software-properties-common && \
+    add-apt-repository ppa:openjdk-r/ppa && \
+    add-apt-repository ppa:ts.sch.gr/ppa && \
+    apt-get -qq -y install curl
+
+
+RUN apt-get -qq update && \
+    apt-get -y install \
+    docker-compose \
+    sudo \
+    python2.7 \
+    python3-pip \
+    openjdk-8-jdk \
```

Comment:
do you need java 8?

Reply:
When I removed java 8 from here, I kept on getting the following error. I tried various changes while searching the solution on the net but couldn't succeed.  Can you suggest what's the missing dependency it is looking for? openjdk-8 must be installing some dependencies required for ca-certificates.

```
head: cannot open '/etc/ssl/certs/java/cacerts' for reading: No such file or directory
dpkg: error processing package ca-certificates-java (--configure):
 installed ca-certificates-java package post-installation script subprocess returned error exit status 1
dpkg: dependency problems prevent configuration of openjdk-11-jre-headless:amd64:
 openjdk-11-jre-headless:amd64 depends on ca-certificates-java (>= 20190405~); however:
  Package ca-certificates-java is not configured yet.
```